ad-scraper.js

```diff
@@ -49,7 +49,11 @@ function parseHTML(html) {
         });
         adData.VIP.adAttributes.forEach(function(a) {
             var attr = a.localeSpecificValues.en;
-            ad.info[attr.label] = attr.value;
+            if (attr.value){
```

Comment:
Like I said, I think it makes more sense to just have `ad.info[attr.label] = a.machineValue;` here.

Reply:
Are you sure that all items will have an a.machineValue field?